main.sh

```diff
@@ -34,6 +34,18 @@ calcTotal() {   # Calculates total in cart.data
     echo $total
 }
 
+displayCart() {
+    echo '______________ YOUR CART ______________'
+    cat cart.data | while read line; do
```

Comment:
I like this code. So the condition is if the file has a line to read, perform the operations, correct? It iterates through each line of the file until there isn't one, then ends the loop? Good stuff.

Reply:
Spot on - I think this could be used to populate an array, like toppings for example. That way, we can add/remove items without editing any scripts/code.